src/features/commands.ts

```diff
@@ -392,6 +392,25 @@ To integrate it into your editor: https://prettier.io/docs/en/editors.html`,
         }
       });
     }
+  },
+  {
+    words: [`!gender`],
+    help: `reminds users to use gender-neutral language`,
+    handleMessage: msg => {
+      msg.channel.send({
+        embed: {
+          title: "Please use gender neutral language by default",
+          type: "rich",
+          description: `Unless someone has made it clear what gender they are, please do not refer to them with gendered language.
+
+Examples:
+
+- Instead of "she/her/hers" or "he/him/his" use "they/them/theirs"
+- "thanks friend" instead of "thanks man"`,
```

Comment:
A couple of small adjustments: I've been trying to phrase stuff positively ("do this" instead of "don't do this"), and I think it'd be good to mention the "hey guys" example. Thoughts? Happy to iterate on this more

```suggestion
          title: "Please use gender neutral language by default",
          type: "rich",
          description: `Unless someone has made their gender known, please use gender neutral language.

- Instead of "hey guys," try "hey folks", "hey all", or similar
- Use "they/them/theirs" if you aren't sure of someone's pronouns
- "thanks friend" instead of "thanks man"`,
```

Reply:
looks good